x-pack/plugins/security/public/views/management/management.js

```diff
@@ -34,7 +34,7 @@ routes.defaults(/\/management/, {
             'xpack.security.management.securityTitle', {
               defaultMessage: 'Security',
             }),
-          order: 10,
+          order: 100,
```

Comment:
yes @snide requested a change in item order

Reply:
It can flip back if you feel strongly about it. It was actually before we introduced the index page and the thought was that we could direct people to a set page that we were sure people had access to.